tools/docker/corda-all-in-one/corda-v4_8-flowdb/build.gradle

```diff
@@ -0,0 +1,139 @@
+buildscript { //properties that you need to build the project
```

Comment:
@elenaizaguirre Could you please write a few sentences of explanation about why was there a need for a different gradle build file instead of the one that the official Corda kotlin samples repo ships with (if that one was broken/didn't work then that's a perfectly valid explanation but I want to make sure the information is there for future generations to have as well)